t/api/13-influxdb.t

```diff
@@ -67,6 +67,30 @@ $t->get_ok('/admin/influxdb/minion')->status_is(200)->content_is(
 openqa_minion_workers,url=http://example.com active=0i,inactive=1i
 "
 );
+
+subtest 'filter specified minion tasks' => sub {
+    for my $task (qw(obs_rsync_run obs_rsync_update_builds_text)) {
+        $t->app->minion->add_task($task => sub { });
+        my $job_ids = $t->app->minion->enqueue($task);
+    }
+    while (my $tmp_job = $worker->dequeue(0)) { $tmp_job->fail('this is a test'); }
+
+    $t->get_ok('/admin/influxdb/minion')->status_is(200)->content_is(
+        "openqa_minion_jobs,url=http://example.com active=0i,delayed=1i,failed=3i,inactive=1i
+openqa_minion_workers,url=http://example.com active=0i,inactive=1i
+"
+    );
+
+    # Define the filter list, so the failed jobs whose task belong to the list
```

Comment:
```suggestion
    # Define the filter list, so failed jobs for blocklisted tasks will not be
    # counted towards the total of failed jobs, i.e. failed=1i instead of failed=3i
```